data/location/address-data/index.html

```diff
@@ -34,7 +34,8 @@
   updates:
     - December 2022 - Added several address grids in Beaver county (Elk Meadows, Ponderosa,
       Greenville, Adamsville, Sulphurdale).  Made major updates to grids in Utah, Cache, Tooele,
-      and Box Elder Counties.  Renamed 'NSL' to 'North Salt Lake' and 'East Carbon City' to 'East Carbon'.
+      and Box Elder Counties.  Renamed 'NSL' to 'North Salt Lake' and 'East Carbon City' to 'East Carbon'. 
```

Comment:
**suggestion**: I think you'll want this to be on the same line?

Reply:
actually it all looks kinda the same.

Reply:
Yeah, the long line/multiline stuff always throws me for a loop.  I put it on multiple lines for readability, but it seems like it renders the same anyways.

Reply:
yeah that is html for ya